### PR TITLE
Write e2e tests for SF - Functions, Containers and Runtimes

### PR DESCRIPTION
**Pull request by AmFlint**

### CODE REVIEW

File: tests/functions/functions.test.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  let namespace;
+
+  beforeAll(() => {
+    oldCwd = process.cwd();
+    serviceName = getServiceName();
+    api = new Api(apiUrl, scwToken);
+    registryApi = new RegistryApi(scwToken);
+
+    //
+    // tmpDirPath = getTmpDirPath();
```
Comment (Retenodus) [inline tests/functions/functions.test.js]: We should not let all this commented code

File: tests/containers/containers.test.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    }
+  });
+
+  it('should remove registry namespace properly', async () => {
+    const response = await registryApi.deleteRegistryNamespace(namespace.registry_namespace_id);
+    expect(response.status).to.be.equal(200);
+  });
+
+  it('should throw error container directory not found', () => {
+    // TODO
```
Comment (Retenodus) [inline tests/containers/containers.test.js]: what's this todo ?

File: package.json [hunk 3]
```diff
@@ -9,7 +9,10 @@
     "git": "https://github.com/scaleway/serverless-scaleway-functions"
   },
   "scripts": {
-    "test": "jest",
```
Comment (almottier) [inline package.json]: you should update the `"version"` to be able to publish the version on NPM

File: package.json [hunk 4]
```diff
@@ -25,13 +28,25 @@
...
     "serverless.com"
   ],
+  "jest": {
+    "testEnvironment": "node",
+    "testRegex": "(/tests/.*|(\\.|/)(test|spec))\\.js$",
+    "useStderr": true,
+    "setupFilesAfterEnv": [
+      "<rootDir>/tests/setup-tests.js"
+    ],
+    "verbose": true
```
Comment (almottier) [inline package.json]: could you add under jest: `"testPathIgnorePatterns": ["tests/utils", "tests/setup-tests.js"],` to avoid testing unnecessary files

**Pull request merged**